frontend/src/services/api/DataHelper.ts

```diff
@@ -0,0 +1,22 @@
+export default class DataHelper {
```

Comment:
Since this class contains a single static function, we aren't gaining much by using it. Consider exporting transformNumber directly from this file.

It might actually make more sense to put this in [ui/utils.ts](https://github.com/codeforboston/jobhopper/blob/933b0a9c01866354fc3495c61fb4a13d188aaa52/frontend%2Fsrc%2Fui%2Futils.ts) since it is not specific the API and is only used by UI components